lib/datadog/tracing/sampling/span/rule_parser.rb

```diff
@@ -22,11 +22,15 @@ class << self
             # @return [Array<Datadog::Tracing::Sampling::Span::Rule>] a list of parsed rules
             # @return [nil] if parsing failed
             def parse_json(rules)
+              return nil unless rules
+
               begin
                 list = JSON.parse(rules)
               rescue => e
-                Datadog.logger.warn("Error parsing Span Sampling Rules `#{rules.inspect}`: "\
-                                    "#{e.class.name} #{e.message} at #{Array(e.backtrace).first}")
+                Datadog.logger.warn(
+                  "Error parsing Span Sampling Rules `#{rules.inspect}`: "\
+                                                      "#{e.class.name} #{e.message} at #{Array(e.backtrace).first}"
+                )
```

Comment:
Rubocop 🤪

Reply:
Yeah, sometimes I let it do it's thing, and then come in after and tweak it so it's still "rubocop-acceptable" but does not look positively awful.

I'm still not convinced of the value of many of the rules in rubocop. Having a linter in general is ok, but rubocop is incredibly picky on irrelevant things, to the point of hindering productivity (at least for me).

Reply:
Agreed. We should axe the rules that don't help. It's supposed to be an aid, not a hinderance.

Reply:
I agree, I think there's a bug in Rubocop regarding multi line argument indentation.